src/api/src/test/java/api/AbstractDatabaseServiceTest.java

```diff
@@ -34,6 +37,13 @@ static void cleanup() {
                 .withNetwork(Network.SHARED)
                 .withNetworkAliases("oracledb");
         oracleContainer.start();
+        try (Connection connection = oracleContainer.createConnection("")) {
```

Comment:
could be done with some callback function so it's not done for all containers :)  but for now let's proceed with this